src/irmin-graphql/unix/info.mli

```diff
@@ -0,0 +1,21 @@
+(*
+ * Copyright (c) 2013-2021 Thomas Gazagnaire <thomas@gazagnaire.org>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *)
+
+module Make (I : Irmin.Info.S) : sig
+  include Irmin.Info.S with type t = I.t
+
+  val v : ?author:string -> ('b, Format.formatter, unit, f) format4 -> 'b
+end
```

Comment:
Copied this module from `irmin-unix` to avoid cyclic dependency. We will be able to remove this code in the future.

Reply:
Yeah, I think this `Info` module is one of the trickiest parts of this change since it is useful to all the `.unix` packages but still defined in irmin-unix.

Reply:
It's definitely a weird one. My plan is to put it in an `irmin.unix` subpackage (this will be the new `Irmin_unix`), but even then then I'm considering exposing it via each of the unix subpackages to make the programming interactions more consistent (ie using `Irmin_git_unix.Info` instead of `Irmin_unix.Info`).

Reply:
Cool, I think exposing it in each of the subpackages would be nice, it would make everything more self-contained.